src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -69,9 +70,9 @@ function CommunityCareAppointmentDetailsPage({
 
   return (
     <div>
-      <div className="vads-u-display--block vads-u-padding-y--2p5 vaos-hide-for-print">
-        ‹ <Link to="/">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```

Comment:
It's a little bit odd, but the last link in our breadcrumbs should be to the current page url. So the `to` prop here should point to `/community-care/${id}`. And the others would have slightly different urls.

Reply:
The last breadcrumb link now points to the same current page url but oddly it's inactive.

Reply:
That's normal for the breadcrumbs. I don't remember the reasoning for it, though.